examples/axum-app/templates/_layout.css

```diff
@@ -0,0 +1,56 @@
+{#-
```

Comment:
It's not a valid css comment. ;)

Reply:
Oh wait, now I got it. You normally never inline CSS into HTML, very bad idea.

Reply:
It's mentioned in `_layout.html`, that normally you should not do that. :)

Reply:
Put it in cap. :3